website_2024/themes/PyCon-Israel-Flex/templates/speakers.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% block htmltag %}<html lang="{{ page.lang|default(DEFAULT_LANG) }}">{% endblock %}
+
+{% block meta %}
+{{ super() }}
+
+{% if page.translations -%}
+<link rel="alternate" href="{{ SITEURL }}/{{ page.url }}" hreflang="{{ page.lang }}" />
+{% for p in page.translations %}
+<link rel="alternate" href="{{ SITEURL }}/{{ p.url }}" hreflang="{{ p.lang }}" />
+{% endfor %}
+{% endif %}
+{% endblock %}
+
+{% block title %} &ndash; {{ page.title|striptags|escape }}{% endblock %}
+
+{% block content %}
+<article class="single" style="max-width: 80%;">
```

Comment:
I note that the only difference between this and the `page.html` template used for most other pages is the addition of `style="max-width: 80%;"` in this line.

There is probably a less-self-repeating way to do that, but it makes me ask -- is this maybe a change we want to include for all pages?